code/game/gamemodes/wizard/artifacts.dm

```diff
@@ -351,14 +351,14 @@
 
 /datum/spellbook_artifact/prestidigitation/purchased(mob/living/carbon/human/H)
 	..()
-	H.add_spell(new/spell/targeted/spark)
-	H.add_spell(new/spell/targeted/extinguish)
-	H.add_spell(new/spell/targeted/clean)
-	H.add_spell(new/spell/targeted/unclean)
-	H.add_spell(new/spell/targeted/create_trinket)
-	H.add_spell(new/spell/targeted/cool_object)
-	H.add_spell(new/spell/targeted/warm_object)
-	H.add_spell(new/spell/targeted/color_change)
+	H.add_spell(new/spell/targeted/spark, iswizard = TRUE)
```

Comment:
argname agreement

Reply:
What do you mean?

Reply:
My mistake, I saw you added a new arg to add_spell and thought you meant to add this as on_added, thus your specified argname would not match here